EdgeModuleMessageSample/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Microsoft.Azure.Devices.Client.Samples
+{
+    public class Program
+    {
+        /// <summary>
+        /// This sample demonstrates how to send and receive messages on an Azure edge module.
+        /// </summary>
+        /// ///<remarks>
+        /// For simplicity, this sample sends telemetry messages to the module itself.
+        /// </remarks>
+        /// <param name="args">
+        /// Run with `--help` to see a list of required and optional parameters.
+        /// </param>
+        public static async Task<int> Main(string[] args)
+        {
+            // Parse application parameters
+            Parameters parameters = null;
+            ParserResult<Parameters> result = Parser.Default.ParseArguments<Parameters>(args)
+                .WithParsed(parsedParams =>
+                {
+                    parameters = parsedParams;
+                })
+                .WithNotParsed(errors =>
+                {
+                    Environment.Exit(1);
+                });
+
+            TimeSpan? appRunTime = null;
+            string outputTarget = null;
+            if (parameters.ApplicationRunningTime.HasValue)
+            {
+                Console.WriteLine($"Running sample for a max time of {parameters.ApplicationRunningTime.Value} seconds.");
+                appRunTime = TimeSpan.FromSeconds(parameters.ApplicationRunningTime.Value);
+                outputTarget = parameters.OutputTarget;
```

Comment:
Should this be inside the if block?

Reply:
we are defining a default value for the output target, if block would not be necessary.

Reply:
But you won't set it if the app run time doesn't have a value.

Why is this tied to that parameter?